hypothesis-python/src/hypothesis/errors.py

```diff
@@ -175,3 +175,12 @@ class RewindRecursive(Exception):
 
     def __init__(self, target):
         self.target = target
+
+
+class SmallSearchSpaceWarning(HypothesisWarning):
+    """Indicates that an inferred strategy does not span the search space
+    in a meaningful way, for example by only creating default instances."""
+
+
+class MissingExtraWarning(HypothesisWarning):
+    """Indicates that an extra package should be installed to improve functionality."""
```

Comment:
See below for details, but I think we can delete this.

Reply:
Yes. I somehow (mistakenly, stupidly) convinced myself that the extras were separate installs, but since they are not this is pointless.